teams/teams-ps/teams/Clear-CsOnlineTelephoneNumberOrder.md

```diff
@@ -0,0 +1,94 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+online version: https://docs.microsoft.com/powershell/module/teams/clear-csonlinetelephonenumberorder
+Module Name: MicrosoftTeams
+title: Clear-CsOnlineTelephoneNumberOrder
+schema: 2.0.0
+manager: mreddy
+author: jiaych
```

Comment:
is this a github username?

Reply:
No, that's the authors Microsoft alias - the GitHub alias is TristanChen-msft - can you please update the metadata accordingly?